q.rd

```diff
@@ -295,57 +296,56 @@ tables may change at any time without prior warning.
               <values nullLiteral="-1"/>
         </column>
     </table>
-    
-    <data id="import_plant_basic_table">
+
+    <data id="import_planet_basic">
```

Comment:
If you remove the _table ending I would suggest doing that with all other tables as well for consisteny reasons. I actually thought the data id had to be the same as the table id but looks like that was a wrong assumption on my part.